judo-ui-typescript-rest-axios/src/main/resources/data-axios/relationServiceImpl.ts.hbs

```diff
@@ -28,6 +28,26 @@ export class {{ serviceRelationName relation }}Impl extends JudoAxiosService imp
   {{/ if }}
 {{/ if }}
 
+{{# if relation.isExportable }}
+    {{# if relation.isCollection }}
```

Comment:
We can use `and` instead of 2 `if`s.

Reply:
Should I rewrite it in the other places as well?

Reply:
[997e71a](https://github.com/BlackBeltTechnology/judo-ui-typescript-rest-template/pull/53/commits/997e71a34bc2b024e29cc9bd46716041867c6cb4)